test/RelayHubPenalizations.test.ts

```diff
@@ -469,44 +475,33 @@ contract('RelayHub Penalizations', function ([_, relayOwner, relayWorker, otherR
         gasPrice: relayCallArgs.gasPrice,
         to: relayHub.address,
         value: relayCallArgs.value,
-        // @ts-ignore
-        chainId: 1,
         data: encodedCall
-      })
+      }, transactionOptions)
 
       transaction.sign(Buffer.from(relayCallArgs.privateKey, 'hex'))
       return transaction
     }
 
     async function getDataAndSignatureFromHash (txHash: string, chainId: number): Promise<{ data: string, signature: string }> {
-      // @ts-ignore
-      const rpcTx = await web3.eth.getTransaction(txHash)
-      // eslint: this is stupid how many checks for 0 there are
-      // @ts-ignore
-      // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
-      if (!chainId && parseInt(rpcTx.v, 16) > 28) {
+      const rpcTx: any = await web3.eth.getTransaction(txHash)
+      const vInteger = parseInt(rpcTx.v, 16)
+      if (chainId == null && vInteger > 28) {
```

Comment:
does this work with chainId==null? you throw for vInteger>28 AND for vInteger <=28

Reply:
This is the exact translation of the old code without the ts-ignore. I don't know if it works but it is only used in this test file so I am not concerned with it at all.